kpi/tests/api/v2/test_api_submissions.py

```diff
@@ -285,13 +311,13 @@ def test_list_submissions_limit(self):
 
     def test_list_submissions_not_shared_other(self):
         self._log_in_as_another_user()
-        response = self.client.get(self.submission_url, {"format": "json"})
+        response = self.client.get(self.submission_list_url, {"format": "json"})
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
 
     def test_list_submissions_shared_other(self):
```

Comment:
probably should be broken out into a new issue, but the method name doesn't make it clear what's being tested